src/Compilers/CSharp/Portable/Compilation/CSharpCompilerDiagnosticAnalyzer.cs

```diff
@@ -36,7 +36,15 @@ internal override ImmutableArray<int> GetSupportedErrorCodes()
                         // We don't support configuring WRN_ALinkWarn. See comments in method "CSharpDiagnosticFilter.Filter" for more details.
                         continue;
 
+                    case (int)ErrorCode.WRN_UnreferencedField:
```

Comment:
@bbarry Hi Bill, can you give me a bit more detail on what you said? I am not sure whether I understood what you are saying.